be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -123,6 +135,7 @@ StatusOr<DriverState> PipelineDriver::process(RuntimeState* runtime_state) {
         _first_unfinished = _new_first_unfinished;
 
         if (sink_operator()->is_finished()) {
+            cancel(runtime_state);
```

Comment:
why cancel?

Reply:
> why cancel?

The function name `cancel` is fuzzy, maybe we can rename sth like `finish_all_operators`?